src/web-page.ts

```diff
@@ -1,7 +1,7 @@
 export class WebPage {
 
     public protocol : string;
-    public headers : any;
+    public headers : Object;
```

Comment:
Good, but have you tested something like `{[property : string] : string}` instead of `Object`?